src/script/standard.cpp

```diff
@@ -89,21 +89,44 @@ static constexpr bool IsSmallInteger(opcodetype opcode)
     return opcode >= OP_1 && opcode <= OP_16;
 }
 
+/** Valid number of keys in a multisig are [1,20]. */
+static constexpr bool IsValidMultisigKeyCount(unsigned int n_keys)
+{
+    return n_keys > 0 && n_keys <= MAX_PUBKEYS_PER_MULTISIG;
+}
+
+static bool GetMultisigKeyCount(opcodetype opcode, valtype data, unsigned int& count)
+{
+    if (IsSmallInteger(opcode)) {
+        count = CScript::DecodeOP_N(opcode);
+    } else {
```

Comment:
Doesn't this need to check what `opcode` is ?

Reply:
`data` would be empty if `opcode` is not a `PUSHDATA` and therefore `count` would be `0` and we'd return `false`. Would an explicitly check be preferable ?